forms/consts.py

```diff
@@ -0,0 +1,7 @@
+import settings
+import re
+import hashlib
+
+HASH = lambda x, y: hashlib.md5(x+y+settings.NONCE_SECRET).hexdigest()
+IS_VALID_EMAIL = lambda x: re.match(r"[^@]+@[^@]+\.[^@]+", x)
+EXCLUDE_KEYS = ['_gotcha', '_next', '_subject', '_cc']
```

Comment:
I think this could be merged into helpers.py (but that's only because I get more confused the more files a project has), what do you think?